src/main/java/org/ambraproject/rhino/service/impl/ConfigurationReadServiceImpl.java

```diff
@@ -42,6 +47,21 @@
   @Autowired
   private GitInfo gitInfo;
 
+  private String hostname = "unknown";
+
+  private Date startTime;
+
+  @PostConstruct
+  public void init() {
+
+    startTime = new Date();
```

Comment:
Looks like, once initialized, `startTime`'s value will never change.  If so, you can remove setting its value in `init()`, and make it `final`:

```
private final Date startTime = new Date();
```